ddtrace/profiling/_periodic.py

```diff
@@ -141,16 +142,17 @@ def run(self):
                     raise
 
 
-def PeriodicRealThread(*args, **kwargs):
+def PeriodicRealThreadClass():
```

Comment:
A capitalised function name could be misleading, but if this formatting is already adopted elsewhere I'm fine with it

Reply:
Perhaps one could do
~~~ python
class PeriodicRealThread():
    def __new__(cls):
        ...

Reply:
I think it's fine as it's meant to be returning a `class`.